environments/nodejs/server.js

```diff
@@ -90,18 +92,37 @@ app.all('/', function (req, res) {
         }
         res.status(status).send(body);
     }
-    try {
-        //
-        // Customizing the request context
-        //
-        // If you want to modify the context to add anything to it,
-        // you can do that here by adding properties to the context.
-        //
-        userFunction(context, callback);
-    } catch(e) {
+
+    //
+    // Customizing the request context
+    //
+    // If you want to modify the context to add anything to it,
+    // you can do that here by adding properties to the context.
+    //
+
+    let functionProm;
+    if (userFunction.length === 1) { // One argument (context)
+        // Make sure their function returns a promise
+        functionProm = Promise.resolve(userFunction(context));
+    } else { // 2 arguments (context, callback)
```

Comment:
Thanks for adding this and preserving compatibility.  Could you keep one example in the old style so it can be tested?  Maybe just make a copy of the old `hello.js` as `hello-callback.js`?